agent/agents/postgres/pgstatstatements/pgstatstatements.go

```diff
@@ -113,15 +113,24 @@ func getPGVersion(q *reform.Querier) (pgVersion float64, err error) {
 	return strconv.ParseFloat(v, 64)
 }
 
+func getPgStatVersion(q *reform.Querier) (pgVersion float64, err error) {
+	var v string
+	err = q.QueryRow(fmt.Sprintf("SELECT /* %s */ extVersion from pg_extension where pg_extension.extname = 'pg_stat_statements'", queryTag)).Scan(&v)
+	if err != nil {
+		return
+	}
+	return strconv.ParseFloat(v, 64)
```

Comment:
Are you sure we can parse the version to float?
Wouldn't a semver comparison make more sense?

Reply:
for sure, that was borrowed from the old code. ill fix to semver

Reply:
done
however had to append '.0' to the query result because postgres returns two-component version